gramps/gen/datehandler/_date_nl.py

```diff
@@ -147,17 +149,17 @@ class DateDisplayNL(DateDisplay):
 
     calendar = (
         "", "juliaans", "hebreeuws",
-        "franse republiek", "persisch", "islamitisch",
+        "frans republikeins", "persisch", "islamitisch",
         "zweeds" )
 
-    _mod_str = ("", "voor ", "na ", "rond ", "", "", "")
+    _mod_str = ("", "voor ", "na ", "omstreeks ", "", "", "")
 
     _qual_str = ("", "geschat ", "berekend ")
 
     _bce_str = "%s v. Chr."
 
     formats = (
-        "JJJJ-MM-DD (ISO)", "Numerisch DD/MM/JJ", "Maand Dag, Jaar",
+        "JJJJ-MM-DD (ISO)", "Numerisch DD/MM/JJJJ", "Maand Dag, Jaar",
```

Comment:
I believe this should be "Numeriek".

Reply:
Indeed, very well found!
Please, change it into "Numeriek".